LanguageFeatures/Extension-types/static_analysis_extension_types_A17_t01.dart

```diff
@@ -26,6 +26,6 @@ extension type V1(int id) {}
 extension type V2<T1, T2 extends num?>(T1 id) {}
```

Comment:
(Would comment on line 16, but Github won't do that.)

I'm afraid there's a bunch of `@assertion`s that need to be updated here, because the description in the feature specification was updated. 

But, as you mentioned somewhere, this PR will make changes such that the tests will work with the current as well as the upcoming semantics, so maybe that's just according to plan, and these updates will be performed in a later PR?

The corresponding change would be needed in the `@description` as well, but that could of course also be done later.

Please just keep a TODO about this somewhere if it is working as intended that we still have the old `@assertion`s and `@description`s.